src/pykiso/test_result/assert_step_report.py

```diff
@@ -219,8 +222,8 @@ def _prepare_report(test: unittest.case.TestCase, test_name: str) -> None:
         test_description = test_method.__doc__ or ""
         ALL_STEP_REPORT[test_class_name]["test_list"][test_name] = {
             "description": test_description,
-            "steps": [],
-            "unexpected_errors": [],
+            "steps": [[]],
```

Comment:
Can we refine the content type of the list here?

Reply:
Added a type hints for ``ALL_STEP_REPORT[test_class_name]["test_list"][test_name]``